### ISSUE TITLE
Changing resolver args using middleware is not possible

### ISSUE THREAD

**Issue by alaarihan** [Urigo/graphql-modules#1438]
Hi,

I created a middleware to change the resolver args but it always return the original args in the resolver function so I think the problem is "next" function don't pass the params to the resolver function

this is my middleware
```
function({ root, args, context, info }, next) {
  args = { limit: 1 }
  return next(root, args, context, info);
};
```

**Comment by adamsultan7** [comment]
I found that it's possible if I did it like this
```
function(ctx, next) {
  ctx.args = { limit: 1 }
  return next();
};
```

**Comment by kamilkisiela** [comment]
We released a new major version (v1.0) of GraphQL Modules. Here's the [migration guide](https://graphql-modules.com/docs/next/recipes/migration).

The new version solves the issue, so I'm closing the thread.

Please try to upgrade. If you can't and still want to use the legacy version, please open a new issue.
